routing/probability_bimodal_test.go

```diff
@@ -309,6 +309,20 @@ func TestIntegral(t *testing.T) {
 	}
 }
 
+// TestSpecialCase tests a value combination found by fuzz tests.
+func TestSpecialCase(t *testing.T) {
```

Comment:
This name is too generic and gives very little info about what's being tested here...maybe add a link to https://github.com/lightningnetwork/lnd/issues/9085?

Reply:
Totally, I changed it and added some description.